components/Main.js

```diff
@@ -28,20 +26,21 @@ const Main = () => {
   const setInfo = () => {
     const data = {
       title: state.title,
-      description: state.description
-    }
-    dispatch(setInfoThunk(data))
-  }
+      description: state.description,
+    };
+    dispatch(setInfoThunk(data));
+  };
 
   return (
     <div className={styles.content}>
-      <CustomHeader />
       <div className={styles.inputs}>
         <label>Tittle:</label>
         <input
           type="text"
           className={styles.title}
-          onChange={(e) => dispatch({ type: "SET_TITLE", text: e.target.value })}
+          onChange={(e) =>
+            dispatch({ type: "SET_TITLE", text: e.target.value })
```

Comment:
Constants such as SET_TITLE might be better to move to enums (or its javascript equivalent)